contents/resources/dariah-de-publikator/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: DARIAH-DE Publikator Tutorial
+date: 2019-12-01
+authors:
+  - dariah-de
+contributors:
+editors:
+categories:
+  - dariah
+tags:
+  - research-infrastructures
+  - dh
+  - metadata
+  - open-science
+  - sustainability
+  - repositories-and-collections
+  - open-access
+abstract:
+"This tutorial explains the fundamentals of the DARIAH-DE Publikator, the entry point for importing collections and research data into the DARIAH-DE Repository, which allows you to prepare, manage, and finally import your collections into the DARIAH-DE Repository using your favourite internet browser.
+The tutorial contains guides for users as well as technical documentation. The user guides are  conveniently organized into three sections: Collections, Log in with a DARIAH-DE Account or with a Federation Account, and Publishing with the DARIAH-DE Publikator. Every section has examples to explain different aspects of the DARIAH-DE Publikator."
+
+type: training module
+remoteUrl: https://repository.de.dariah.eu/doc/services/submodules/publikator/docs/index.html
+remotePublicationDate: 2019-12-01
+---
+
+This tutorial explains the fundamentals of the DARIAH-DE Publikator, the entry
+point for importing collections and data into the DARIAH-DE Repository, which
+allows you to prepare, manage, and finally import your collections into the
+DARIAH-DE Repository using your favourite internet browser. The DARIAH-DE
+Repository provides the ability to store research data and enrich them with
+metadata. Through the use of persistent identifiers, a permanent
+machine-readable reference is ensured and findable via a generic search.
+Likewise, the repository makes it possible to store data collections sustainably
+and securely. With the help of the DARIAH-DE Publikator, research data can
+easily be imported into the DARIAH-DE Repository via a graphical user interface
+and tagged with metadata. These research data can be described as a collection
+in the DARIAH-DE Collection Registry and are then identified in the DARIAH-DE
+Generic Search. The tutorial is conveniently divided into several sections. The
+user-guides give step-by-step instructions on how to use the DARIAH-DE
+Publikator. This is presented in three parts: Collections, Log in with a
+DARIAH-DE Account or with a Federation Account, and Publishing with the
+DARIAH-DE Publikator. The technical documentation explains in detail how the
+DARIAH-DE Publikator works by presenting its service APIs.
+
+###Note: This resource is currently only available in German
```

Comment:
this resource _is_ in english, so i don't know what's this. 

also, make sure you preview things because this is published as: 

<br/>
<img width="551" alt="Скриншот 2020-05-25 08 59 07" src="https://user-images.githubusercontent.com/5304839/82787296-6796a480-9e66-11ea-85a5-31ddf4287def.png">

you have to have a space after the "###" for it to count as a markdown heading